src/components/Checkout/InContext/InContextCheckout.vue

```diff
@@ -214,6 +214,9 @@ export default {
 		},
 		isCorporateCampaignPage() {
 			return this.$route.path.substring(0, 4) === '/cc/';
+		},
+		showLcaLoanMessage() {
+			return this.isCorporateCampaignPage && cookieStore.get('lcaid') && this.promoFund?.promoPrice && this.lcaLoanPrice > 0;
```

Comment:
Looks like GH is showing an error for line length. Also it needs to be `this.cookieStore`

Reply:
yeahhh I made a small change & forgot to lint here, I just corrected that